app/controllers/api/messaging.php

```diff
@@ -1669,7 +1675,12 @@
 
         $response
             ->setStatusCode(Response::STATUS_CODE_CREATED)
-            ->dynamic($subscriber, Response::MODEL_SUBSCRIBER);
+            ->dynamic(
+                $subscriber
+                    ->setAttribute('userId', $user->getId())
+                    ->setAttribute('userName', $user->getAttribute('name')),
```

Comment:
Let's do the `setAttribute` calls before the call to `dynamic`, it will be easier to read

Reply:
fixed this